quickstarts/flutter-android/config.yml

```diff
@@ -0,0 +1,45 @@
+slug: flutter-android
+description: |
+  ## Comprehensive monitoring quickstart for Flutter Android
+  
+  Monitor your Flutter applications and analyze performance bottlenecks while operating in a live development environment. You can use this dashboard to visualize your web user experience with your application rendition.
+  
+  ## Why monitor Flutter Android?
+  Monitoring app performance helps you to take a preventative approach to enhance quality rather than waiting for a crash, issue, or user complaint to occur. The Newrelic’s Android and  Flutter agent allows you to track important metrics including network failures and http response, slow launches, tracking custom traces, and more.
```

Comment:
can use `etc.`, instead of `and more`.
@rahulbasu-nr , requesting your expertise